homeassistant/components/permobil/sensor.py

```diff
@@ -0,0 +1,224 @@
+
+
+SENSOR_DESCRIPTIONS: tuple[PermobilSensorEntityDescription, ...] = (
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x,
+        item=BATTERY_STATE_OF_CHARGE,
+        key="state_of_charge",
+        name="Permobil Battery Charge",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.BATTERY,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x,
+        item=BATTERY_STATE_OF_HEALTH,
+        key="state_of_health",
+        name="Permobil Battery Health",
+        icon="mdi:battery-heart-variant",
+        native_unit_of_measurement=PERCENTAGE,
+        device_class=SensorDeviceClass.BATTERY,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x,
+        item=BATTERY_CHARGE_TIME_LEFT,
+        key="charge_time_left",
+        name="Permobil Charge Time Left",
+        icon="mdi:battery-clock",
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x,
+        item=BATTERY_DISTANCE_LEFT,
+        key="distance_left",
+        name="Permobil Distance Left",
+        native_unit_of_measurement=UnitOfLength.KILOMETERS,
+        device_class=SensorDeviceClass.DISTANCE,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x,
+        item=BATTERY_INDOOR_DRIVE_TIME,
+        key="indoor_drive_time",
+        name="Permobil Indoor Drive Time",
+        native_unit_of_measurement=UnitOfTime.HOURS,
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+    ),
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x * BATTERY_ASSUMED_VOLTAGE,
+        item=BATTERY_MAX_AMPERE_HOURS,
+        key="max_watt_hours",
+        name="Permobil Battery Max Watt Hours",
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
+    ),
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x * BATTERY_ASSUMED_VOLTAGE,
+        item=BATTERY_AMPERE_HOURS_LEFT,
+        key="watt_hours_left",
+        name="Permobil Watt Hours Left",
+        native_unit_of_measurement=UnitOfEnergy.WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL,
+    ),
+    PermobilSensorEntityDescription(
+        value_fn=lambda x: x,
+        item=BATTERY_MAX_DISTANCE_LEFT,
+        key="max_distance_left",
+        name="Permobil Full Charge Distance",
+        native_unit_of_measurement=UnitOfLength.KILOMETERS,
+        device_class=SensorDeviceClass.DISTANCE,
+        state_class=SensorStateClass.MEASUREMENT,
```

Comment:
```suggestion
```

Reply:
Why is this not a measurement?

The sensor represents the distance the wheelchair can go if it is fully charged, it is expected to change based on the battery health and the average consumption of the wheelchair.

Reply:
Because this sensor is a prediction

Reply:
Okay so I shouldn't set the state_class at all then?